radio/src/gui/480x272/lcd.cpp

```diff
@@ -280,7 +280,7 @@ void drawTimerMode(coord_t x, coord_t y, int32_t mode, LcdFlags att)
 {
   if (mode >= 0) {
     if (mode < TMRMODE_COUNT)
-      return lcdDrawTextAtIndex(x, y, STR_VTMRMODES, mode, att);
+      lcdDrawTextAtIndex(x, y, STR_VTMRMODES, mode, att);
```

Comment:
return is needed yes, I have nothing against 2 lines here, but with `{ }` in the else part if they are in the if part!